pathd/path_pcep.c

```diff
@@ -314,32 +331,49 @@ DEFUN_NOSH(pcep_cli_pcc, pcep_cli_pcc_cmd,
 }
 
 DEFUN(pcep_cli_pce_opts, pcep_cli_pce_opts_cmd,
-      "pce ip A.B.C.D [port (1024-65535)] [sr-draft07]",
+      "pce [ip A.B.C.D | ipv6 X:X::X:X] [port (1024-65535)] [sr-draft07]",
```

Comment:
Same here

Reply:
Both options are used in FRR, so I'll just leave it as is.